src/map/conquest_data.cpp

```diff
@@ -90,7 +90,12 @@ uint8 ConquestData::getRegionOwner(REGION_TYPE region) const
         return regionControls[regionNum].current;
     }
 
-    ShowError(fmt::format("Invalid conquest region passed to function ({})", regionNum));
+    // This can be called for Latent effects that work outside of conquest areas eg. Dynamis/Limbus
+    if (regionNum > (uint8)REGION_TYPE::LIMBUS)
```

Comment:
This appears to just be suppressing a warning but still returning 0.  Can you guarantee latents will work in this case?

Reply:
Yes, the conditions are different, regionControls.size() is 18 (Tavnazia is the last one) but the Latent for "Areas outside of nation's control" (Expects a 0) will continue to work up to 27 (Limbus). The error is meaningless for 19-27, as it is the expected use case.

Reply:
Are you looking for this check to be reverted before this can be merged? Just let me know, so I understand how to proceed.

Reply:
Please let me know what you want me to do with this PR.

Reply:
this is what I thought up of. There's some "valid" conquest regions that are outside of the conquest data but can still activate latents

```c++
uint8 ConquestData::getRegionOwner(REGION_TYPE region) const
{
    uint8 regionNum = static_cast<uint8>(region);

    // Handle some conquest regions that don't have conquest info as non-error
    // TODO: Do Sandoria/Bastok/Windurst count as "owned by" themselves, no one, or some other state where latents don't work?
    switch (region)
    {
        case REGION_TYPE::TAVNAZIA:
        case REGION_TYPE::SANDORIA:
        case REGION_TYPE::BASTOK:
        case REGION_TYPE::WINDURST:
        case REGION_TYPE::JEUNO:
        case REGION_TYPE::DYNAMIS:
        case REGION_TYPE::TAVNAZIAN_MARQ:
        case REGION_TYPE::PROMYVION:
        case REGION_TYPE::LUMORIA:
        case REGION_TYPE::LIMBUS:
            return NATION_TYPE::NATION_BEASTMEN;
        default:
            if (regionNum < regionControls.size())
            {
                return regionControls[regionNum].current;
            }
    }

    ShowError(fmt::format("Invalid conquest region passed to function ({})", regionNum));
    return 0;
}
```

Reply:
Updated my comment, I double checked why tavnazia is listed twice and one was a legitimate region with conquest data

Reply:
Thanks for your suggestions, I appreciate you spending time looking into this. Sorry, I've wanted to finish it but also want to make sure I take the time to properly test it and I've been very busy the last week. Will update soon.